modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/scope/SelectScope.es.js

```diff
@@ -0,0 +1,258 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import getCN from 'classnames';
+import {addParams, fetch} from 'frontend-js-web';
+import React, {useEffect, useRef, useState} from 'react';
+
+import {SCOPE_TYPES} from '../../utils/constants.es';
+import {sub} from '../../utils/language.es';
+import SelectScopeModal from './SelectScopeModal.es';
+
+const SelectScope = ({
```

Comment:
Can we add docs to what this component renders and it's purpose?

Also I think it'd be more appropriate to name this `ScopeSelect` in the same way `MultiSelect` is named. So the description is the prefix. And then any nested components would be `ScopeSelect*` like `ScopeSelectModal`.